<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>22</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>지역입력 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBox1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(197, 220, 190)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weathr_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>19</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(32, 97, 97)</string>
    </property>
    <property name="text">
     <string>날씨조회</string>
    </property>
   </widget>
   <widget class="QLabel" name="area_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 말랑말랑 Bold</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(43, 39, 68)</string>
    </property>
    <property name="text">
     <string>날씨 조회 지역 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weaimg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69))</string>
    </property>
    <property name="text">
     <string>날씨 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_l1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼매직체</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(56, 71, 69)</string>
    </property>
    <property name="text">
     <string>- 23.5˚</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ytemp_l">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>어제와의 날씨 비교</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 214, 218)</string>
    </property>
    <property name="text">
     <string>체감온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 214, 218)</string>
    </property>
    <property name="text">
     <string>미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 214, 218)</string>
    </property>
    <property name="text">
     <string>초미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scens_l">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(0, 65, 65)</string>
    </property>
    <property name="text">
     <string>26.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(0, 65, 65)</string>
    </property>
    <property name="text">
     <string>좋음</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust_2">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(0, 65, 65)</string>
    </property>
    <property name="text">
     <string>나쁨</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
